Addition of note (#8892)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/microsoft-defender-atp/attack-surface-reduction.md

GUID: `BE9BA2D9-53EA-4CDC-84E5-9B1EEEE46550`

> > [!Note]

Reviewer: ```suggestion
> [!NOTE]
```

> > The rule **Block executable content from email client and webmail** is also reffered as following. 

Reviewer 2: "referred to" seems a bit misplaced or imprecise. Maybe "has the following separate descriptions, depending on which application you use:" can be applied.
```suggestion
> The rule **Block executable content from email client and webmail** has the following separate descriptions, depending on which application you use:
```
I also defer to JohanFreelancer9 for a final conclusion.

Reviewer 2: (My first thought was to simply correct the phrasing as it is: "is also referred to as follows:", but on second thought, it felt imprecise.)

> > The rule **Block executable content from email client and webmail** has the following separate descriptions, depending on which application you use:

Reviewer: ```suggestion
> The rule **Block executable content from email client and webmail** has the following alternative descriptions, depending on which application you use:
```

> > Intune (Configuration Profiles): Execution of executable content (exe, dll, ps, js, vbs, etc.) dropped from email (webmail/mail client) (no exceptions)

Reviewer: ```suggestion
- > Intune (Configuration Profiles): Execution of executable content (exe, dll, ps, js, vbs, etc.) dropped from email (webmail/mail client) (no exceptions).
```

Reviewer 2: I believe you meant to add a bullet point list inside the Note blob, which needs to be on the inside of the indent marker, instead of outside:
```suggestion
> - Intune (Configuration Profiles): Execution of executable content (exe, dll, ps, js, vbs, etc.) dropped from email (webmail/mail client) (no exceptions)
```

> > Endpoint Manager: Block executable content download from email and webmail clients

Reviewer: ```suggestion
- > Endpoint Manager: Block executable content download from email and webmail clients.
```

Reviewer 2: Same here, it needs to be on the inside of the indent marker, instead of outside:
```suggestion
> - Endpoint Manager: Block executable content download from email and webmail clients
```

> > Group Policy: Block executable content from email client and webmail

Reviewer: ```suggestion
- > Group Policy: Block executable content from email client and webmail.
```

Reviewer 2: And again, it needs to be on the inside of the indent marker, instead of outside:
```suggestion
> - Group Policy: Block executable content from email client and webmail.
```

> > [!Note]
> > The rule **Block executable content from email client and webmail** has the following separate descriptions, depending on which application you use:
> > Intune (Configuration Profiles): Execution of executable content (exe, dll, ps, js, vbs, etc.) dropped from email (webmail/mail client) (no exceptions)
> > Endpoint Manager: Block executable content download from email and webmail clients
> > Group Policy: Block executable content from email client and webmail

Reviewer 2: This is just to show that it is possible to include all the changes in one change, by dragging the [+] sign down along the lines to be changed in source view. Feel free to merge this one in place of all the individual lines:
```suggestion
> [!NOTE]
> The rule **Block executable content from email client and webmail** has the following alternative descriptions, depending on which application you use:
> - Intune (Configuration Profiles): Execution of executable content (exe, dll, ps, js, vbs, etc.) dropped from email (webmail/mail client) (no exceptions).
> - Endpoint Manager: Block executable content download from email and webmail clients.
> - Group Policy: Block executable content from email client and webmail.
```

Reviewer 2: (Please note: I recommend committing **either A)** the _individual_ suggestions **or B)** _this single one_ .

Author: Thank for the tip, but before coming here, I have committed all the above changes.

> - > Intune (Configuration Profiles): Execution of executable content (exe, dll, ps, js, vbs, etc.) dropped from email (webmail/mail client) (no exceptions).
> - > Endpoint Manager: Block executable content download from email and webmail clients.
> - > Group Policy: Block executable content from email client and webmail.

Reviewer 2: ```suggestion
> - Intune (Configuration Profiles): Execution of executable content (exe, dll, ps, js, vbs, etc.) dropped from email (webmail/mail client) (no exceptions).
> - Endpoint Manager: Block executable content download from email and webmail clients.
> - Group Policy: Block executable content from email client and webmail.
```

Reviewer 2: I guess it was unclear how I tried to explain it earlier. This is at least how it needs to be to work as a bullet point list inside the Note blob.

Reviewer: Thanks @illfated. My bad.
